File: Searching Algorithms/Unordered Linear Search/linear-search.java
import java.util.Scanner;
public class linear{
    public static void main(String[] args){
        int n,i;
        Scanner s=new Scanner(System.in);
        System.out.println("Enter the number of elements you are going to enter into the array:");
        n=s.nextInt();
        int a[]=new int[n];
        System.out.println("Enter the elements:");
        for(i=0;i<n;i++){
            a[i]=s.nextInt();
        }
        int search;
        System.out.println("Enter the element you wish to find:");
        search=s.nextInt();
        int flag=0,pos=0;
        for(i=0;i<n;i++){
            if(a[i]==search){
                flag=1;
                pos=i+1;
                break;
            }
        }
        if(flag==0){
            System.out.println("Element has not been found in the array");
        }
        else{
            System.out.println("Element has been found at position: "+pos+" in the array");
        }
    }
}
        
        

File: Searching Algorithms/Unordered Linear Search/linear-search.c
#include <stdio.h>
#include <stdlib.h>
int main(){
  int arr[1000],i,n,num,flag=0;
  printf("Enter the number of elements that are to be entered into the array: \n");
  scanf("%d",&n);
  printf("Enter the elements in the array: \n");
  for(i=0;i<n;i++){
    scanf("%d",&arr[i]);
  }
  printf("Enter the element that has to be searched in the array: \n");
  scanf("%d",&num);
  for(i=0;i<n;i++){
    if(arr[i]==num){
      flag=1;
      printf("The element has been found at %d position",i+1);
      return 0;
    }
  }
  if(flag==0){
    printf("The element is not present in the array \n");
  }

return 0;
  
}
